public/templates/albums/show.html

```diff
@@ -5,7 +5,12 @@
                 Downloads
             </a>
             <ul class="dropdown-menu" ng-show="album.is_downloadable == 1">
-                <li bindonce ng-repeat="format in album.formats"><a target="_blank" bo-href="format.url"><span bo-text="format.name"></span> <small bo-text="'(' + format.size + ')'"></small></a></li>
+                <li bindonce ng-repeat="format in album.formats" ng-hide="isEncoding">
```

Comment:
"is encoding" is inconsistent here with the terminology used in the database ("is in progress"). Any particular reason for this choice?